File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/OpenAiFileObject.kt
package com.sdkwork.ai

data class OpenAiFileObject(
    val id: String? = null,
    val object: String? = null,
    val bytes: Int? = null,
    val filename: String? = null,
    val purpose: String? = null,
    val status: String? = null,
    val deleted: Boolean? = null,
    val createdAt: Int? = null,
    val expiresAt: Int? = null,
    val statusDetails: String? = null
)

File: sdkwork-ai-sdk-csharp/Models/OpenAiFileObject.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class OpenAiFileObject
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public int? Bytes { get; set; }
        public string? Filename { get; set; }
        public string? Purpose { get; set; }
        public string? Status { get; set; }
        public bool? Deleted { get; set; }
        public int? CreatedAt { get; set; }
        public int? ExpiresAt { get; set; }
        public string? StatusDetails { get; set; }
    }
}
